### DISCUSSION TITLE
Base IP address overrules VPN IP address

### DISCUSSION THREAD

**Discussion by myhobby2** [magicbug/Cloudlog#3073] [General]
**This is for reporting bugs with the code, if this issue is regarding setting up on your local server please use the forum see readme for details**

**Describe the bug**
I can access my machines cloudlog webserver both local and via zerotier. But cloudlog webaccess via Zerotier causes an issues and falls back to my local (192.168.1.40) ip-address instead of the assigned zerotier IP address (192.168.192.40). 
Zerotier works 100% because I can login via the commandline everywhere in the world on this machine (which is running cloudlog).

**To Reproduce**
You need an instance with 2 ip-addresses to access this platform

My exact example 

http://192.168.192.40/cloudlog/index.php/user/login -> http://192.168.192.40/cloudlog/index.php/user/login (no problem)
http://192.168.192.40/cloudlog/ -> http://192.168.1.40/cloudlog/index.php/user/login (no vpn connection when external)
http://192.168.1.40/cloudlog/ -> http://192.168.1.40/cloudlog/index.php/user/login (no problem)
http://192.168.1.40 -> http://192.168.1.40/cloudlog/index.php/user/login (via index.php in www-root (no problem)

**Expected behaviour**
I expect that the base IP address used will remain unchanged but in the 2nd line it does.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by magicbug** [comment]
I've converted this to a discussion as it is not a code bug per say, its a configuration and I'm not sure what to suggest!

**Reply by myhobby2** [reply]
Because I don't know the specs, I can't tell either. Nevertheless... I would expect the IP address to remain the same when calling a web-based solution. In my case, the app responds to the other IP address on the platform, resulting in not able to access my logbook from my mobile phone on the road. Maybe other VPN solutions have the same isssue btw
